components/CategorySection.tsx

```diff
@@ -42,25 +43,33 @@ export function CategorySection(props: {
     setCategories(newCategories);
   };
 
+  const renderCategories = () => {
+    return isLoadingCategories ? (
+      <p>Loading</p>
+    ) : errorCategories ? (
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+        }}
+      >
+        <h1>Error</h1>
+        <p>{unknownErrorToString(errorCategories)}</p>
+      </div>
+    ) : (
+      <></>
+    );
+  };
+
   return (
     <div>
       <p className={styles.header}>Categories</p>
       <CategoryHelpers />
       <AddCategoryButton projectId={projectId} />
       <div>
         {isLoadingCategories && <p>Loading</p>}
```

Comment:
We are showing two of the same loader now

Reply:
fixed